library/core/src/intrinsics.rs

```diff
@@ -55,11 +55,9 @@
 #![allow(missing_docs)]
 
 use crate::marker::DiscriminantKind;
-#[cfg(not(bootstrap))]
 use crate::marker::Tuple;
 use crate::mem;
 
-#[cfg(not(bootstrap))]
```

Comment:
The doctests in here require #105356 , which I believe is not yet included in the bootstrap compiler this is using. This particular bump can be fixed by not removing this  just yet. I'll add more granular `#[cfg(not(bootstrap))]` in the future, which should prevent this from reoccuring